### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by shourjo-h** [ReVanced/revanced-discussions#652] [General] [answered]
Can you guys please change the apk's logo? so that it may look like real YouTube? I am not talking about the proper Youtube logo, make it something like that, kind of YouTube vanced?

**Comment by reisxd** [comment]
Three words: Cease and Desist.

**Reply by ghost** [reply]
Nah... Logo can also improve without necessarily be the same to the one of Google.

**Reply by reisxd** [reply]
> Nah... Logo can also improve without necessarily be the same to the one of Google.

>> make it something like that, kind of YouTube vanced?

Edit: And also
>> so that it may look like real YouTube?

**Reply by ghost** [reply]
> > Nah... Logo can also improve without necessarily be the same to the one of Google.
> 
> > > make it something like that, kind of YouTube vanced?

Well...waiting for poll.

**Comment by Ushie** [comment, marked as answer]
> Can you guys please change the apk's logo? so that it may look like real YouTube? I am not talking about the proper Youtube logo, make it something like that, kind of YouTube vanced?

There will be a logo poll later on
